packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -75,14 +82,29 @@ class OpPerfTelemetry {
 		private readonly deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>,
 		logger: ITelemetryLoggerExt,
 	) {
-		this.logger = ChildLogger.create(logger, "OpPerf");
+		console.log(
+			"Creating OpPerfTelemetry\n",
+			"this.deltaManager.clientDetails: ",
+			JSON.stringify(this.deltaManager.clientDetails),
+			"this.deltaManager.connectionManager: ",
+			JSON.stringify((this.deltaManager as unknown as any).connectionManager),
+			"this.deltaManager.deltaManager.connectionManager: ",
+			JSON.stringify((this.deltaManager as unknown as any).deltaManager?.connectionManager),
+		);
+		const samplingConfiguration = new Map<string, number>([["OpRoundtripTime", 5]]);
+		this.logger = ChildLogger.create(logger, "OpPerf", undefined, samplingConfiguration);
```

Comment:
This is how we configure the new sampling; when instantiating a `ChildLogger`, pass a new parameter with the mapping from event name to sampling rate (5 in this case meaning "1 in 5").